doc/source/science_guide/sg_dynamics.rst

```diff
@@ -398,19 +438,24 @@ An elliptical yield curve is used, with the viscosities given by
 
 .. math::
    \zeta = {P(1+k_t)\over 2\Delta},
+   :label: zeta
 
 .. math::
    \eta  = e_g^{-2} \zeta,
+   :label: eta
 
 where
 
 .. math::
    \Delta = \left[D_D^2 + {e_f^2\over e_g^4}\left(D_T^2 + D_S^2\right)\right]^{1/2}.
+   :label: Delta
+
+When the deformation :math:`\Delta` tends toward zero, the viscosities tend toward infinity. To avoid this issue, :math:`\Delta` needs to be limited and is replaced by :math:`\Delta^*` in equation :eq:`zeta`. Two methods for limiting :math:`\Delta` (or for capping the viscosities) are available in the code. If the namelist parameter ``capping`` is set to 1., :math:`\Delta^*=max(\Delta, \Delta_{min})` :cite:`Hibler79` while with ``capping`` set to 0., the smoother formulation  :math:`\Delta^*=(\Delta + \Delta_{min})` of :cite:`Kreyscher00` is used. 
```

Comment:
This explanation for capping is helpful -- I haven't been able to keep the options straight.  Maybe these could be set as capping = 'max' or 'sum' in namelist?  Just a suggestion, and it doesn't have to be in this PR.

Reply:
Capping was specified as a string  before but Till modified it to speed up the code. It is now a real and if statements have been removed.

Reply:
I see.  We should keep the real parameter in the dynamics code, but we could still have the namelist values be meaningful, and add an if block in the initialization to set it to 0 or 1 for later use.  E.g. keep capping = c0 or c1 but make the namelist be viscosity_cap = 'max' or 'sum'.